### ISSUE TITLE
Incorrect absence reported

### ISSUE THREAD

**Issue by moinkhanif** [microverseinc/microverse-issue-tracker#268]
Hey team,

On both the 19th and 20th of February 2020, there's an invalid absence reported on my account. Please have a check. I did attend the standup meeting.

**Comment by Keqi** [comment]
Hi @moinkhanif,

**Feb 19** - Our system did not see you at the Standup meeting. People who were present are:

- Daniel Sokil
- Barak Ben-Hamu
- Ramsey Njire
- Jude Kajura

Is there any chance you mistakenly joined the other room? Did you have some connectivity issues on that day?

**Feb 20** - Standup meeting wasn't registered at all. Is there any chance you joined the meeting alone and nobody else was there?

**Comment by moinkhanif** [comment]
Hey Maciej, Thanks for getting back to me on this. No, I usually join the meeting through the dashboard. I was having issues with my computer and had to reinstall windows. I did notice that I wasn't signed in to zoom on the first day. Could that have caused this issue? And yes No one was present on that day I remember leaving the chat soon. But on 20th I joined the room as soon as the link a little early because I wanted to talk to my team soon and end the day due to my sleep issues.
